m4_sessions/1710-inning/group_three.md

```diff
@@ -1,10 +1,28 @@
-Group Three
+## Group Three
 
-Session Title: 
-Description: Why should students come to this session? 
+### Session Title:
 
-Session Outcomes: What will students leave with or gain from this session?
+Global Decentralization (work-in-progress)
 
-Session Outline:  Include the flow of the session and the time for each section.
+### Description: Why should students come to this session?
 
-Names of group members 
+We feel that this is something that will be impacting more and more people as time goes on, and is a really untapped market for development opportunities. Also great for us to be aware of it as developers. 
+
+### Session Outcomes: What will students leave with or gain from this session?
+
+Students will leave with a better understanding of decentralization, how quickly and dramatically our world will change as a whole.  Students will have knowledge of different industry sectors.
+
+### Session Outline:  Include the flow of the session and the time for each section.
+
```

Comment:
Give students about 10 minutes to do an icebreaker or introduce each other.  They won't know everyone in the session.